omod/src/main/webapp/pages/login.gsp

```diff
@@ -41,6 +37,15 @@ ${ ui.includeFragment("referenceapplication", "infoAndErrorMessages") }
             updateSelectedOption();
         });
         
+        jQuery('#login-form').submit(function(e) {
+        	var sessionLocationVal = jQuery('#sessionLocationInput').val();
+        	
+        	if (!sessionLocationVal) {
```

Comment:
Line 28 has three checks to confirm that location is not selected. Your change has one check. Did you just ignore the other two checks?